src/components/subscriptions/licenses/LicenseManagementTable/LicenseManagementTableBulkActions.jsx

```diff
@@ -181,6 +184,70 @@ const LicenseManagementTableBulkActions = ({
     );
   };
 
+  const getTotalToRevoke = () => {
+    // If not revoking all users, return the number of selected users that are revocable
+    if (!allUsersSelected) {
+      return usersToRevoke.length;
+    }
+
+    // If there are no filters applied, return the number of revocable users
+    if (activeFilters.length === 0) {
+      return activatedUsersCount + assignedUsersCount;
+    }
+
+    // If any filter besides the status filter is applied, we cannot get an accurate count of revocable users
+    if (activeFilters.find(filter => filter.name !== 'statusBadge')) {
+      return null;
+    }
+
+    // If only the status filter is applied and it includes revoked users, substract the number
+    // of revoked users from the table item count
+    const activeFilter = activeFilters[0];
+    if (activeFilter.name === 'statusBadge' && activeFilter.filterValue.includes(REVOKED)) {
+      return tableItemCount - revokedUsersCount;
+    }
+
+    return tableItemCount;
+  };
+
+  const getTotalToRemind = () => {
```

Comment:
nit: This could likely utilize `useMemo` to only compute the value when necessary rather the re-calculate it on each render.